modules/amxBidAdapter.js

```diff
@@ -0,0 +1,279 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+import { parseUrl, deepAccess, _each, formatQS, getUniqueIdentifierStr, triggerPixel } from '../src/utils.js';
+
+const BIDDER_CODE = 'amx';
+const SIMPLE_TLD_TEST = /\.co\.\w{2,4}$/;
+const DEFAULT_ENDPOINT = 'https://prebid.a-mo.net/a/c';
```

Comment:
When I test the test bidders this endpoint is returning 404

VM321:1 POST https://prebid.a-mo.net/a/c 404

Reply:
Hi @GLStephen -- did you use the tag id from above? e.g:

```
{
  bidder: 'amx',
  params: {
    testMode: true,
    tagId: 'cHJlYmlkLm9yZw'
  }
}
```

For test ads to work, we need both `testMode: true` and that tagId

Reply:
@nickjacob I saw that, and I used it, but the endpoint 404d, it wasn't rejected

Reply:
![image](https://user-images.githubusercontent.com/232630/86290031-b7316400-bbba-11ea-9b54-c784177dbf96.png)

Reply:
With this: I get no 404, but a noBid

bids: [{
                bidder: 'amx',
                params: {
                    testMode: 'true',
                    tagId: 'bfeh23'
                }
            }]

With this I get a 404:

bids: [{
                bidder: 'amx',
                params: {
                    testMode: true,
                    tagId: 'bfeh23'
                }
            }]

Reply:
can you use `cHJlYmlkLm9yZw` as the tagId, e.g. `tagId: 'cHJlYmlkLm9yZw'` ? It has to be that exact value (that's our test site placement)

Reply:
using that you should see our test ad (e.g. on hello_world.html):

![image](https://user-images.githubusercontent.com/1192267/86290624-be0ca680-bbbb-11ea-82eb-b7cb58668f80.png)

Reply:
@nickjacob ok, that is working, your test placement should be in the docs so that a person working on integrating your bidder can confirm proper functionality. I'm also seeing that endpoint even with the test params failing just about every other time. Any idea why?

Reply:
@nickjacob now it seems to be working every time, I would make sure it is functional and in the docs, so that users can verify operation

Reply:
We nobid on `testMode: "true"` because we check that it's a boolean (if it's present) [here](https://github.com/prebid/Prebid.js/blob/b5e03eab7e2ba7e172e606f0ae2bc29b99cea6df/modules/amxBidAdapter.js#L138)

If you copy the full bid object I pasted above it should work

Reply:
what error do you get when it fails? 404 or something else? I'll make sure to add the test information to the docs thanks!

Reply:
what error do you get when it fails? 404 or something else? Locally I get a 200 each time

I'll make sure to add the test information to the docs thanks!

Reply:
@nickjacob that worked and I set it back to the boolean